spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -140,6 +140,20 @@ sealed class SpdxExpression {
     fun isValid(strictness: Strictness = Strictness.ALLOW_CURRENT): Boolean =
         runCatching { validate(strictness) }.isSuccess
 
+    /**
+     * Return true if [choice] is a valid license choice for this SPDX expression. This is the case, if [choice] does
+     * not contain the [OR operator][SpdxOperator.OR], and if the [licenses][SpdxSingleLicenseExpression] contained in
```

Comment:
I believe the "does not contain OR" part is more or less an implementation detail. How about "This can only be the case if [choice] does not offer a choice itself and if the [licenses]..."

Reply:
I think "to contain OR" and "offer a choice" are identical by definition, but I'm fine with changing the sentence.

Reply:
Done.